src/components/ChatWindow.tsx

```diff
@@ -21,11 +21,13 @@ type Props = {
   title?: string;
   subtitle?: string;
 };
+
 type State = {
   message: string;
   messages: Array<Message>;
   customerId: string;
   conversationId: string | null;
+  API_BASE_URL: string;
```

Comment:
state/prop values should not be formatted like a constant, also probably should move this from `state` to `props` if that doesn't break things